### PR TITLE
Fix fallback logic for view settings

### LINKED ISSUE #165: Old-style settings do not work anymore
Since 55709c6 (Expose all supported settings in the default settings (#159)), released as 2.2.1 the old-school settings don't work, for example `origami_auto_close_empty_panes`

```
        auto_close = view.settings().get("origami_auto_close_empty_panes", False)
        auto_close = self.settings().get("auto_close_empty_panes", auto_close)
```

Since 55709c6 `auto_close_empty_panes` is always defined, so this code will never fall back to the original settings name.

### PR DESCRIPTION
**Pull request by kaste**
Fixes #165

### CODE REVIEW

File: origami.py [hunk 1]
```diff
@@ -761,8 +761,10 @@ def on_close(self, view):
 
     def on_pre_close(self, view):
         # Read from global settings for backward compatibility
-        auto_close = view.settings().get("origami_auto_close_empty_panes", False)
-        auto_close = self.settings().get("auto_close_empty_panes", auto_close)
+        auto_close = (
+            view.settings().get("origami_auto_close_empty_panes")
+            or self.settings().get("auto_close_empty_panes")
+        )
```
Comment (rchl) [inline origami.py:767]: We've created a fix at the same time (#167) :)

Your handles a setting that I didn't but the logic is not correct, I think. We want to use `origami_auto_close_empty_panes` if it's a boolean, even if it is false.

Comment (kaste) [reply origami.py:767]: :LOL: That was the lazy approach.  We can have a `is None` check of course.  Wait a sec.

**Review by rchl — APPROVED** [review]

**Comment by kaste** [comment]
Just pushed the non-lazy approach.  Take either the first commit, or squash them.

**Comment by rchl** [comment]
Thanks

**Pull request merged**